CONTRIBUTING.md

```diff
@@ -56,6 +56,25 @@ release branch. For example, if the most recent branch is `release-0.2`, and the
 development for v0.3.0, a bug fix that merged to `master` that also affects `v0.2.x` may be considered for backporting
 to `release-0.2`. We generally do not accept PRs against older release branches.
 
+### Proposal process (CAEP)
+
+The Cluster API Enhacement Proposal is the process this project uses to adopt new features, or changes to the APIs.
+
+- The template, and accepted proposals live under `docs/proposals`.
+- A proposal SHOULD be introduced and discussed during the weekly community meetings.
+- A proposal MUST be submitted first to the community using a collaborative writing platform, preferably Google Docs.
+  - When using Google Docs, share the document with the SIG mailing list.
+
+### Experiments
+
+Proof of concepts, code experiments, or other initiatives can live under the `exp` folder and behind a feature gate.
+
+- Experiments SHOULD ship with a CAEP marked as experimental.
+- Experiments SHOULD not modify any of the publicly exposed APIs (e.g. CRDs).
+- Experiments follow a strict lifecycle: Alpha -> Beta -> GA.
+  - Alpha-stage experiments SHOULD not be enabled by default.
+  - Inactive or outdated experiments will be deprecated and removed from the codebase.
```

Comment:
Refer to K8s deprecation policy for the deprecation rules?

Reply:
do you have a link? I couldn't find one?

Reply:
https://kubernetes.io/docs/reference/using-api/deprecation-policy/

Reply:
Not sure if I'd link to the API deprecation policy for experiments